### ISSUE TITLE
Question about learning rate

### ISSUE THREAD

**Issue by norips** [2toinf/X-VLA#70]
Hello,

Thank you for this project!

Regarding learning rate, related to #55 , VLM and soft prompt learning rate should be lower than the model learning rate. Does it mean that https://github.com/2toinf/X-VLA/blob/8dfe7910e461b8fc43b0d0ac941b81defb6e2548/train.py#L123-L128

should be modified to use the same LR as the soft prompt one like:
```diff
@@ -121,7 +129,7 @@ def build_optimizer(model: XVLA, lr: float, weight_decay: float, betas=(0.9, 0.9
     exclude = set(map(id, vlm_params + soft_prompt_params + action_params))
     transformer_core_params = [p for p in model.parameters() if id(p) not in exclude]
     param_groups = [
-        {"name": "vlm", "params": vlm_params, "lr": 0.0, "weight_decay": weight_decay},
+        {"name": "vlm", "params": vlm_params, "lr": lr * lr_coef_soft, "weight_decay": weight_decay},
         {"name": "transformer_core", "params": transformer_core_params, "lr": 0.0, "weight_decay": weight_decay},
         {"name": "soft_prompts", "params": soft_prompt_params, "lr": lr * lr_coef_soft, "weight_decay": weight_decay}
```

Should the same be done for `transformer_core` ?

**Comment by 255isWhite** [comment]
build_optimizer is just an initialization of the optimizer; our learning rate schedule is applied [here](https://github.com/2toinf/X-VLA/blob/8dfe7910e461b8fc43b0d0ac941b81defb6e2548/train.py#L154)(train.py L154)

**Comment by norips** [comment]
Thanks for clearing it up!
